index.js

```diff
@@ -8,51 +8,73 @@ const INT_MASK = MASK;
 
 const SHORT_MASK = new BN("0xFFFF", 16);
 
+const getCtor = val =>
```

Comment:
This allows the module to compare the constructor in a way that `instanceof` does not allow. For example my `BN` can be different from your `BN` and `instanceof` breaks down in that case.